config/import-control.xml

```diff
@@ -159,8 +159,8 @@
     <!-- until https://github.com/checkstyle/checkstyle/issues/3095 -->
     <allow pkg="antlr"/>
     <allow class="com.puppycrawl.tools.checkstyle.DetailAstImpl"/>
-    <allow class="java.nio.charset.StandardCharsets" />
```

Comment:
expalin this update.

if it is unrelated to your changes but non-required config - please send removal in separarte PR.

Reply:
@romani I added `<allow class="java.nio.charset.StandardCharsets" />` in my previous PR https://github.com/checkstyle/checkstyle/pull/8317. Later on I removed the code which required this import, but forgot to remove from `import-control.xml`.

Should I keep it in this PR or create a seperate PR for this?

Reply:
Keep in this PR but please move this change to separate commit, commit should have "minor:" prefix. This PR Will have 2 commits

Reply:
Okay @romani

Reply:
Please swap commits in order, functional fix should be last commit in your branch

Reply:
done @romani